css-canvas-medical/styles.css

```diff
@@ -0,0 +1,241 @@
+  margin: 0 auto;
+  font-family: 'Tiempos Headline', sans-serif;
+  font-size: 24px;
+}
+
+h3 {
+  margin: 15px auto;
+  font-size: 16px;
+  text-transform: uppercase;
+}
+
+p {
+  line-height: 30px;
+}
+
+ul {
+  padding: 0;
+  margin: 0;
+  list-style: none;
+}
+
+header {
+  padding: 20px;
+  border-bottom: 1px solid rgb(196, 196, 196);
+}
+
+footer {
+  background-color: rgb(77, 77, 77);
+}
+
+a {
+  color: inherit;
+  text-decoration: none;
+}
+
+.container {
+  margin: 0 auto;
+}
+
+.row {
+  display: flex;
+}
+
+.column-banner {
+  flex-basis: 100%;
+  padding: 0 25px;
+}
+
+.two-column-banner {
+  flex-basis: 50%;
+  padding: 0 25px;
+}
+
+.three-column-banner {
+  flex-basis: 25%;
+  padding: 0 25px;
+}
+
+.four-column-banner {
+  flex-basis: 25%;
+  padding: 0 25px;
+}
+
+.banner-logo {
+  display: flex;
+  align-items: center;
+}
+
+.justify-flex-end {
+  flex-basis: 75%;
+  padding: 0 25px;
+  margin: 30px 25px;
+  display: flex;
+  align-items: center;
+  color: rgba(255, 249, 249, 0.801);
+  font-weight: normal;
+  justify-content: flex-end;
+}
+
+.logo-blue {
+  height: 30px;
+}
+
+.nav-menu {
+  margin: 0 auto;
+  color: rgb(160, 160, 160);
+  display: flex;
+  align-items: center;
+  list-style: none;
+}
+
+.get-started-button-two {
+  display: flex;
+  justify-content: flex-end;
+}
+
+.banner {
+  padding: 35px;
+}
+
+.background {
+  position: relative;
+  padding: 50px 10px;
+}
+
+.background-image {
+  background-image: url('images/hero.jpg');
+  position: absolute;
+  top: 0;
+  right: 0;
+  bottom: 0;
+  left: 0;
+  background-size: cover;
+  background-position: center;
+  background-repeat: no-repeat;
+  filter: brightness(100%) contrast(70%) saturate(75%);
+}
+
+.banner-content {
+  padding: 35px;
+  position: relative;
+  color: white;
+  max-width: 590px;
+}
+
+.banner-content p {
+  color: white;
+  line-height: 22px;
+  font-size: 16px;
+}
+
+.get-started-button {
+  padding: 20px;
+  color: white;
+  display: inline-block;
+  text-decoration: none;
+  background-color: rgb(33, 57, 87);
+  border-radius: 3px;
+  height: 3.5rem;
+  font-size: 20px;
+  font-weight: 100;
+}
+
+.practices {
+  margin: 0 auto;
+  padding: 10px 100px 147px;
+  background-color: #efefef;
+  text-align: center;
+}
+
+.offices {
+  font-size: 17px;
+}
+
+.offices > li {
+  margin-bottom: 50px;
+}
+
+.learn-more-banner {
+  padding: 3rem 0 1.5rem;
+  border-top: 1px solid rgb(231, 233, 236);
+}
+
+.learn-more-banner h2 {
+  margin-bottom: 0;
+}
+
+.company-info {
+  padding: 7.25rem 0 6.75rem;
+  color: white;
+  background-color: rgb(82, 83, 81);
+}
+
+.bottom-logo {
+  height: 30px;
+  margin-bottom: 2.75rem;
+}
+
+.footer-links-one li {
+  margin-bottom: 1.25rem;
+  font-size: 14px;
+}
+
+.footer-links-two,
+.footer-links-one,
+.privacy-and-contact-links {
+  font-size: 14px;
+}
+
+.company-links {
+  position: relative;
+  left: 1000px;
+  bottom: 229px;
+}
+
+.product-links {
+  position: relative;
+  left: 1200px;
+  bottom: 340px;
+}
```

Comment:
Using position relative with these properties is not the appropriate way to position your links, please use the grid system with flex properties.